### PR TITLE
SUI Color Panel

### PR DESCRIPTION
**Pull request by jcger**

### CODE REVIEW

File: components/atom/colorPanel/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# AtomColorPanel
+
+> A color panel is a container with a color on its background.
+
+## Installation
+
+```sh
+$ npm install @schibstedspain/sui-atom-colorPanel --save
```
Comment (davidbarna) [inline components/atom/colorPanel/README.md]: Do we use uppercase for package names ???

File: components/atom/colorPanel/src/index.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    'sui-AtomColorPanel',
+    `sui-AtomColorPanel-${type}`
+  )
+
+const getStyle = ({opacity}) => ({
+  opacity
+})
+
+const AtomColorPanel = ({children, ...props}) => (
+  <div className={getClassNames(props)} style={getStyle(props)}>
```
Comment (davidbarna) [inline components/atom/colorPanel/src/index.js]: Is the opacity supposed to be for the component or for its background ?

**Review by davidbarna — APPROVED** [review]

File: components/atom/colorPanel/README.md (hunk 1 above)
Comment (jcger) [reply components/atom/colorPanel/README.md]: nope, its the autogenerated readme that does this and I didn't check it. will be fixed

File: components/atom/colorPanel/src/index.js (hunk 2 above)
Comment (jcger) [reply components/atom/colorPanel/src/index.js]: background, thanks!

Comment (davidbarna) [reply components/atom/colorPanel/src/index.js]: Then I request the change !! :-)

**Comment by JoanClaret** [comment]
Not sure if we need a component to assign a background-color.
i would propose to define the variables in `sui-theme`, and do not develop a React component.
but maybe i'm missing something!

```scss
$c-color-panel-canvas: $c-white !default;
$c-color-panel-default: color-variation($c-gray, 4) !default;
$c-color-panel-base: color-variation($c-gray, 3) !default;
$c-color-panel-contrast: $c-black !default;
$c-color-panel-corporate: $c-primary !default;
$c-color-panel-highlight: color-variation($c-accent, 4) !default;
```

**Comment by nguasch** [comment]
@JoanClaret it make sense. I think that @ julian-gernun has the idea to merge `Color Image` component with this one, but maybe one good option would be had the `Color Panel` as a mixin in `sui-theme` and use it in `Image Panel` component

**Comment by jcger** [comment]
Closing this because we will create a mixing as color panel and use it in a "sui-atom-panel" component which will be "image-panel" + "color-panel"

**Pull request closed without merging**